docs/semgrep-app/sso.md

```diff
@@ -61,4 +61,21 @@ To set up SSO:
 
 If you encounter issues during the setup process, reach out to [support@r2c.dev](mailto:support@r2c.dev) for assistance.
 
-<MoreHelp />
+### Setting up SAML SSO with Azure Active Directory
+
+Set up Semgrep as an Enterprise App with SAML SSO using Azure's Active Directory service by following these steps:
+
+1. Enable **Enterprise App** in **Azure Active Directory** with SAML SSO selected. Follow the instructions in [Enable single sign-on for an enterprise application](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso).
```

Comment:
Which section or sections of the linked document should we read to find this information?

Reply:
outdated - resolving as "addressed in subsequent commits by me"